Software/Tuning Motion Magic PID.md

```diff
@@ -0,0 +1,34 @@
+# Tuning Motion Magic PID
+
+Follow these steps to tune the Motion Magic PID for your robot:
+
+## Prerequisites
+- Ensure you are connected to the robot via its Wi-Fi.
+- Open the `constants.py` file to adjust the PID constants.
+
+## Steps
+
+1. **Set Initial PID Constants**  
+    Set all PID constants (`kP`, `kI`, `kD`, `kF`, etc.) to `0` in `constants.py`.
+
+2. **Tune `kS` (Static Gain)**  
+    - Gradually increase the `kS` value.  
+    - Deploy the code after each adjustment.  
+    - Stop increasing `kS` once the mechanism begins to move slightly.  
+      > **Note:** Keep this value as low as possible while ensuring the mechanism moves.
+
+3. **Tune `kG` (Gravity Compensation)**  
```

Comment:
At some point add that you should be moving the mechanism to be straight forward